docs/outlook/microsoft-graph.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Use the Microsoft Graph REST API from an Outlook add-in
+description: Learn how to use the Outlook mail REST API from an Outlook add-in with Microsoft Graph.
+ms.date: 01/30/2025
+ms.topic: how-to
+ms.localizationpriority: medium
+---
+
+# Use the Microsoft Graph REST API from an Outlook add-in
+
+With the Outlook JavaScript API, you can retrieve the properties of messages and appointments and run operations on these items in your add-in. However, there may be instances where the data you need isn't exposed through the API. For example, your add-in may need to implement single sign-on or identify messages in a user's mailbox that originated from the same sender. To get the information you need, use the [Outlook mail REST API](/graph/api/resources/mail-api-overview) through [Microsoft Graph](/graph/overview).
+
+## Get started
+
+Before you can make calls to the Microsoft Graph API, you must first perform the following:
+
+1. [Register your add-in in the Azure portal](/graph/auth-register-app-v2).
```

Comment:
https://learn.microsoft.com/en-us/office/dev/add-ins/develop/enable-nested-app-authentication-in-your-add-in covers this already. This link is for the older getAccessToken OBO approach which is more complicated. So let's not recommend that direction.